node/node.go

```diff
@@ -313,7 +307,7 @@ func (node *Node) addPendingStakingTransactions(newStakingTxs staking.StakingTra
 		}
 	}
 	node.pendingStakingTxMutex.Unlock()
-	utils.Logger().Info().Int("length of newStakingTxs", len(newStakingTxs)).Int("totalPending", len(node.pendingTransactions)).Msg("Got more staking transactions")
+	utils.Logger().Info().Int("length of newStakingTxs", len(newStakingTxs)).Int("totalPending", len(node.pendingStakingTransactions)).Msg("Got more staking transactions")
```

Comment:
do we have staking transaction?

Reply:
Staking is not supported in s3. But somehow the initial staking txns skeleton code is in s3. It's no-op so shouldn't be a problem.